site/docs/v1/tech/client-libraries/netcore.adoc

```diff
@@ -37,11 +37,97 @@ include::docs/v1/tech/client-libraries/_static-patch-note.adoc[]
+
+* FusionAuth Blazor Server Application.
+* FusionAuth MVC Application.
+* FusionAuth Web API Application.
+
+
+=== Requirements
+
+* A FusionAuth installation. You can link:/docs/v1/tech/installation-guide/[install FusionAuth locally, or sign up for a hosted account].
+* .NET Core 7.0 SDK or higher.
+* Visual Studio 2022 for Mac, version 17.6 and above (optional).
+* Visual Studio 2022 for Windows, version 17.6 and above (optional).
+
+
+=== Installing the Templates
+
+FusionAuth .NET templates are available on link:https://www.nuget.org/packages/FusionAuth.Templates/[NuGet]. You can install them by running the following command in your terminal.
+
+```bash
+dotnet new install FusionAuth.Templates::1.0.0
+```
+
+When installed successfully, the templates will be available in the .NET CLI and Visual Studio. The installation is the same for both Windows and macOS.
+
+=== Using the Templates
+
+To help you set up a valid application in FusionAuth, we have created a .NET FusionAuth setup script, available on https://github.com/FusionAuth/fusionauth-example-client-libraries/tree/main/dotnet[GitHub]. Follow the instructions in the Github repo to set up FusionAuth for your project. Note that the setup script is designed to run on a newly installed FusionAuth instance with only one user and no tenants other than `Default`. To use the setup script on a FusionAuth instance that does not meet these criteria, you may need to modify the script.
+
+
+Some templates will ask for a FusionAuth Client Secret when initializing a new project. Use a non-sensitive secret from a local FusionAuth installation. If you use the setup script to set up your FusionAuth instance, the Client Secret will be `change-this-in-production-to-be-a-real-secret`.
+
+Do not use a production or sensitive Client Secret in the template prompts, as it is stored in the `Appsettings.json.development` file, and will be available in plain text and committed to your repo. To provide the Client Secret to your application in production, you should use one of the secure methods https://learn.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-7.0&tabs=windows[recommended by Microsoft]. The platform-independent hierarchical path for the Client Secret setting is `FusionAuth__ClientSecret`.
+
+==== .NET CLI
+
+To create a new project from a template, run the following command.
+
+```bash
+dotnet new [template-name] [options]
+```
+
+Where `[template-name]` is the name of the template you want to use from one of the following:
+
+- `fusionauthblazorserver` creates a new Blazor Server application with FusionAuth authentication and authorization.
+- `fusionauthmvcwebapp` creates a new MVC application with FusionAuth authentication and authorization.
+- `fusionauthwebapi` creates a new Web API application with FusionAuth authentication and authorization.
+
+Use `[options]` to provide your FusionAuth URL and FusionAuth Application Client Id. The following options are available:
```

Comment:
There should be an option for a port since the application is required to run on a specific port in the FusionAuth settings.

Reply:
nice add, thx